### DISCUSSION TITLE
Forward port to WG Client

### DISCUSSION THREAD

**Discussion by thesosguy** [wg-easy/wg-easy#213] [Q&A]
HI All,

I have tested out all the WG gui's and by far this is the smoothest and easiest to maintain. It works amazingly well.

tl;dr - I need help with forwarding a public port to a specific WG client ip address.
          wg_server_ip:23000 to a wg openwrt client

I'm using it to essentially tunnel specific ports to a openwrt router as wg client from the public IP of the WG server. 
On my other WG server with out docker that is accomplished by adding an iptable rule as follows. 

`iptables -t nat -A PREROUTING -p tcp --dport 23000 -j DNAT --to-destination 10.8.0.2:5000`
`iptables -t nat -A PREROUTING -p tcp --dport 23001 -j DNAT --to-destination 10.8.0.2:80`

On my existing server it works beautifully, openwrt connects to WG the router is set to port forward.
I'm able to connect to it at <public_ip_of_wg_server>:23000

This is by no means correct but it is where I am stuck. (I am not an iptables expert)
My docker run 100% standard just the relevant info changed.
```
. . . . .
-e WG_POST_UP="iptables -t nat -A DOCKER ! -i docker0 -p tcp -m tcp --dport 23000 -j DNAT --to-destination 10.8.0.2:23000"
-e WG_POST_UP="iptables -t nat -D DOCKER ! -i docker0 -p tcp -m tcp --dport 23000 -j DNAT --to-destination 10.8.0.2:23000"
. . . . .
```
I know it is not the most secure but it is just a DVR listening on those ports that would otherwise have open ports to it anyways im just trying to secure the local network. Basically I want to recreate a product remotewinbox.com but just not for Mikrotik devices.

Thanks for any help, and thank you Emile
